src/agent/src/health_management.c

```diff
@@ -29,7 +27,7 @@ _Bool IsConnectionInfoValid(const ADUC_LaunchArguments* launchArgs)
 {
     _Bool validInfo = false;
 
-    ADUC_ConnectionInfo info = { NULL, NULL };
+    ADUC_ConnectionInfo info = { ADUC_AuthType_NotSet,ADUC_ConnType_NotSet, NULL,NULL,NULL,NULL };
```

Comment:
>ADUC_AuthType_NotSet,ADUC_ConnType_NotSet, NULL,NULL,NULL,NULL  [](start = 33, length = 63)

nit: spacing...

Reply:
run clangformat, cmakeformat.

---
In reply to: [591975275](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591975275) [](ancestors = 591975275)